frontend/integration-tests/views/crud.view.ts

```diff
@@ -167,3 +172,35 @@ export const emptyState = $('.cos-status-box').$('.text-center');
 
 export const errorMessage = $('.alert-danger');
 export const successMessage = $('.alert-success');
+
+export const createNamespacedResourceWithDefaultYaml = async function(resourcetype:string) {
+  await browser.get(`${appHost}/k8s/ns/${testName}/${resourcetype}`);
+  await isLoaded();
+  const createDropdownIsPresent = await createItemButton.isPresent();
+  if (createDropdownIsPresent) {
```

Comment:
I would take common creation code from `crud.scenario.ts` and add it here so we're not duplicating logic.

Reply:
Compared code in crud.scenario.ts with this code block, Moved common code into clickYamlCreation() for reuse.